README.md

```diff
@@ -23,9 +23,10 @@ import '@github/auto-check-element'
 Provide a URL and a CSRF token and the autocheck component will show validation confirmations and validation errors.
 
 The endpoint should respond to POST requests with:
- - a 200 HTTP status code if the provided value if valid.
- - a 422 HTTP status code if the provided value is invalid.
- - a optional error message in the body and a `Content-Type` header with a value of `text/html; fragment`.
+
+- a 200 HTTP status code if the provided value if valid.
+- a 422 HTTP status code if the provided value is invalid.
+- a optional error message in the body and a `Content-Type` header with a value of `text/html; fragment`.
```

Comment:
I think this line should be updated since we don't care about the response body anymore.